### ISSUE TITLE
How to decrypt with PKCS7 padding?

### ISSUE THREAD

**Issue by johnngap** [ricmoo/aes-js#23]
How to decrypt with PKCS7 padding?

thanks.

**Comment by ricmoo** [comment]
I do not currently have a PKCS#7 utility function with this library, but I should certainly add one. I will have it added this week.

**Comment by johnngap** [comment]
Hi ricmoo, Have any update for PKCS#7 utility function?

**Comment by ricmoo** [comment]
Heya! Oh yes! Sorry, I have the code written, I just need to polish off some test cases and example usage. Sorry for the delay, I will get to this shortly.

**Comment by ricmoo** [comment]
I have pushed these changes to gitHub, but not yet published them to NPM. Please try them out. (Commit: https://github.com/ricmoo/aes-js/commit/8d3fc6f350c0270acc5049d6e1b0e0c19d89036e)

Example:

```
var decryptedData = aes.decrypt(ciphertext);
var strippedData = aesjs.padding.pkcs7.strip(decryptedData);
```

The test cases test the happy path. More test cases need to be added in the future to ensure bad data correctly (loudly) fails.

These test cases probably won't get in until after v3.0 which is moving to Uint8Array.

**Comment by johnngap** [comment]
Hi ricmoo, It works perfectly for me.
I can decrypt with PKCS7 padding easily now.
Thanks for your help!!

**Comment by Ruffio** [comment]
@johnngap So this issue can be closed now?

**Comment by ricmoo** [comment]
Heya!

Yup, thanks.

I have published the new version with the PKCS#7 padding functions to npm.
